service/certificate-generator/src/test/java/de/adorsys/psd2/sandbox/certificate/service/CertificateServiceTest.java

```diff
@@ -28,7 +27,9 @@ public void newCertificateCreatesCertAndKey() {
     CertificateRequest certificateRequest = CertificateRequest.builder()
         .authorizationNumber("12345")
         .countryName("Germany")
-        .organizationName("adorsys").build();
+        .organizationName("adorsys")
+        .commonName("Rozetka")
```

Comment:
common name shall correspond to other attributes. Use real values in production, not fake ones

Reply:
applies everywhere in this PR